README.md

```diff
@@ -38,10 +40,8 @@ For more specific topics, join
 TiDB has the following types of groups that are officially supported:
```

Comment:
Do we need this section here?

Reply:
how about removing them after adding the governance of SIG and WGs?

Reply:
OK

Reply:
I think we can leave this section and add the link to the governance file when it is merged.

Reply:
Do you mean to remove the `Governance` section in this file?

Reply:
I think Calvin means that we can replace the governance section with a link. So we do not need to maintain 2 files.

Reply:
Sorry about the confusion. You can refer to TiKV community readme. We can have the section here, but only provides the link to Governance.

Reply:
Correct.

Reply:
But we don't have this link in the current repo 😂

Reply:
We can remove the link text and add it after the governance is merged.